### DISCUSSION TITLE
Pointing the Website's "Community" / Forum section to eg here?

### DISCUSSION THREAD

**Discussion by SeanScherer** [endless-sky/endless-sky#7748] [General]
Hey everyone :) !

The "Forum" Link on the website ( https://endless-sky.github.io/ - bottom left) points to a google groups that seems to be in disuse. 

I think pointing it here might be a better spot? (Alternatively, you could point at the Steam discussions section or whatever they have?).

_Note, there is a issue on the website repository raising this (https://github.com/endless-sky/endless-sky.github.io/issues/23), but I reckoned chances for something to happen are better here :P._

Thanks and Regards, Sean (aka "Righal" in Endless Sky :D)

P.S. To give some background, I only stumbled over this game the last couple days, and I really like it! (ok - the story building and interface, the ship to ship action - not so much :P ... ). 
I found it by chance via a semi-random search that landed me on a list of the "[top 25 Open source games of 2020](https://www.makeuseof.com/tag/open-source-video-games/)", which had this video showcasing the game: https://youtu.be/As1599mECy0 (which I thought was very neat :)  )

**Comment by SeanScherer** [comment]
Ah, and just found this (rooting around a little more): 

Seems the pointer to the forum was removed from the readme (or changed into a "forum archives are here" notice) ->  https://github.com/endless-sky/endless-sky/pull/5743 (with considerable leg work :P).

So I guess the question is not whether to remove the dead link, but whether to replace it, and if yes, where to point it.

(can cross reference this discussion on the website's issue if so wished)

**Comment by quyykk** [comment]
https://github.com/endless-sky/endless-sky.github.io/pull/28 I'll try to finish that PR soonish.

**Reply by SeanScherer** [reply]
Cool, I hadn't realised there was someone "on task" :).

**Comment by Zitchas** [comment]
Welcome to Endless Sky! 

Yes, pointing the forum link to here would be a good idea. 

I would like to keep an "archival" link to the old google groups - it is still kind of accessible (although not particularly active), but something over the years has rendered it inaccessible to anyone who hadn't already visited it prior to google doing something. And even some people who *did* visit it previously can't reach it anymore. But it is to some of us. But yeah, it's a relic of a bygone era at this point.

**Reply by SeanScherer** [reply]
Nice to be welcomed :).

Kinda the wrong place to ask this, but here goes: Does the Discord server/channels have bridged rooms in Matrix by any chance? I'm guessing no (I did try to do a search for them Matrix side, and no luck).

Where would a better spot to ask about this be ? (I'm guessing probably on the Discord server itself? - I do have an account, so I'm game for that :P ).

**Reply by MasterOfGrey** [reply]
It does not, but if you're interested and willing to explain what that is/what's involved/why it's good, then it's possible that bridged rooms are something it *could* have.

**Reply by SeanScherer** [reply]
Ok. I'm a bit lost on the Discord Server as of now, so I'll guess I'll be opening a new Topic here when I do (I even have some people that - may - be willing to do part of the heavy lifting. I'll have to chat :P).

Thanks for the feedback in any case :).

**Comment by SeanScherer** [comment]
If it was up to me, I wouldn't include that on the website. 
It (the link to the zombie google group) is in the Git(hub) repos readme, so anyone that is going to look into adding stuff is highly likely to find it there, and it seems kinda pointless to aim general public / random passersby at it.

But that may be me :P .
